sdcflows/workflows/fit/pepolar.py

```diff
@@ -171,40 +194,38 @@ def init_topup_wf(
     # fmt: on
 
     if not debug:
+        # Roll orientation back to original
+        from_las = pe.Node(ReorientImage(), name="from_las")
+        from_las_fmap = pe.Node(ReorientImage(), name="from_las_fmap")
         # fmt: off
         workflow.connect([
-            (concat_blips, pad_blip_slices, [("out_file", "in_file")]),
-            (pad_blip_slices, topup, [("out_file", "in_file")]),
-            (topup, ref_average, [("out_corrected", "in_file")]),
-            (topup, outputnode, [("out_field", "fmap"),
-                                 ("out_warps", "out_warps")]),
+            (setwise_avg, from_las, [("out_file", "target_file")]),
+            (setwise_avg, from_las_fmap, [("out_file", "target_file")]),
+            (topup, from_las, [("out_corrected", "in_file")]),
+            (from_las, ref_average, [("out_file", "in_file")]),
+            (topup, from_las_fmap, [("out_field", "in_file")]),
+            (topup, outputnode, [("out_warps", "out_warps")]),
+            (from_las_fmap, outputnode, [("out_file", "fmap")]),
         ])
         # fmt: on
         return workflow
 
-    from nipype.interfaces.afni.preprocess import Volreg
     from niworkflows.interfaces.nibabel import SplitSeries
     from ...interfaces.bspline import ApplyCoeffsField
 
-    realign = pe.Node(
-        Volreg(args=f"-base {grid_reference}", outputtype="NIFTI_GZ"),
-        name="realign_blips",
-    )
+    # Separate the runs again, as our ApplyCoeffsField corrects them separatelly
```

Comment:
super nitpick but typo
```suggestion
    # Separate the runs again, as our ApplyCoeffsField corrects them separately
```